apps/storybook/src/HeatmapVisDisplay.stories.tsx

```diff
@@ -98,65 +98,16 @@ WithAxisLabels.args = {
 
 export const WithAnnotation: Story<HeatmapVisProps> = (args) => (
```

Comment:
I still keep one, very simple annotation story for the Heatmap and another for the Line vis.